shortener.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |s|
   s.rubyforge_project         = "shortener"
   s.required_rubygems_version = "> 1.3.6"
   s.add_dependency "rails", ">= 3.0.7"
+  s.add_dependency "voight_kampff", '~>1.0.2'
```

Comment:
@blackbirdco @jpmcgrath is it possible to relax the version of `voight_kampff`, as voight_kampff version 1.0.x depends on rack 1.0, but rails 5 depend on rack 2, which makes shortener cannot be used in rails 5 projects.